src/Database/Connection.php

```diff
@@ -615,7 +615,7 @@ public function isSavePointsEnabled(): bool
     /**
      * Creates a new save point for nested transactions.
      *
-     * @param string|int $name The save point name.
+     * @param string|int $name Save point name or id
      * @return void
      */
     public function createSavePoint($name): void
```

Comment:
I don't know why but these take `string|int` instead of just string. It didn't seem right changing this to `$savePoint` because we pass in the name instead of generating the save point to use elsewhere.